src/xcode/ENA/ENA/Source/Coordinator.swift

```diff
@@ -51,6 +51,7 @@ class Coordinator: RequiresAppDependencies {
 			store: self.store
 		)
 	}()
+	
```

Comment:
Remove newline

Reply:
Maybe it was already removed but I would currently leave it like it is.